src/app/(proper_react)/redesign/(public)/page.tsx

```diff
@@ -14,11 +18,18 @@ export default async function Page() {
   const countryCode = getCountryCode(headers());
   const eligibleForPremium = isEligibleForPremium(countryCode, enabledFlags);
 
+  // request the profile stats for the last 30 days
+  const profileStats = await getProfilesStats(
+    new Date(Date.now() - 30 * 24 * 60 * 60 * 1000),
+  );
+  const oneRepActivations = profileStats?.total_active ?? 0;
+  const scanLimitReached = oneRepActivations > monthlySubscribersQuota;
```

Comment:
In @flozia's PR I saw what I think is the opposite of this behaviour - if OneRep doesn't return how many scans we've already done, it assumes that we've already reached limit. To avoid us accidentally hammering them after we've already crossed the limit.

There's something to be said for the other way around too, but we should at least be consistent, so maybe discuss with him if you'd like to keep it this way :)

Otherwise:

```suggestion
  const oneRepActivations = profileStats?.total_active;
  const scanLimitReached = typeof oneRepActivations === "undefined" || oneRepActivations > monthlySubscribersQuota;
```

Reply:
@codemist Is this on someone's list to actually fix? Otherwise we're going to run into really hard-to-debug issues if we hit the limit, now that this is merged...